### PR TITLE
Limit targets

### PR DESCRIPTION
**Pull request by alealvarez14**
This PR includes:
- A user can only create up to 10 targets

### CODE REVIEW

**Review by diebarral — APPROVED** [review]

File: app/models/target.rb [hunk 1]
```diff
@@ -23,4 +23,9 @@ class Target < ApplicationRecord
   belongs_to :user
 
   validates :title, :radius, :lat, :long, presence: true
+  validate :limit_targets, on: :create
+
+  def limit_targets
+    errors.add(:target, 'Can only create 10 targets') if user.targets.count >= 10
```
Comment (Alxzu) [inline app/models/target.rb]: It would be nice to use `I18n.t` and to make `10` a constant

Comment (alealvarez14) [reply app/models/target.rb]: Fixed!

**Review by Alxzu — APPROVED** [review]
Nice!

File: app/models/target.rb [hunk 2]
```diff
@@ -19,8 +19,16 @@
...
+  MAX_TARGETS = 10
+
   belongs_to :topic
   belongs_to :user
 
   validates :title, :radius, :lat, :long, presence: true
+  validate :limit_targets, on: :create
+
+  def limit_targets
+    return unless user.targets.count >= 10
```
Comment (Alxzu) [inline app/models/target.rb:31]: Leave a space after returns

File: app/models/target.rb [hunk 3]
```diff
@@ -19,8 +19,17 @@
...
```
Comment (GuilleLeopold) [inline app/models/target.rb:31]: Missed constant here, you updated the constant below but not here

**Pull request merged**